File: level 22/classwork/classwork.py
li=[5,8,"apple",True,10,20,4.5,"pear",False,"grape"]
user=int(input("enter a number:"))
if user>=0 and user<len(li):
    print(li[user])
elif user>=len(li)*1 and user <=1:
    print(li[user])
else:
    print(None)



list1=[2,4,6,8,10,12,14,16,18,20]
for i in list1:
    print(i*2)
    print(i/2)

File: level 71/classwork/classwork.js
let programmingLanguages = new Array("C++","Python","C#","Js")
console.log(programmingLanguages.length)
// ვხმარობთ new და Array ქივორდს რითიც ვქმნით სიას. იყენება უფრო კომპლექსურ დავალებებში

let fruits = ["apple","banana","orange","strawberry","pear"]
console.log(fruits.length)
// არ ვიყენებთ ქივორდებს და ისე ვქმნით სიას. უფრო სწრაფი ვერსია

let mixed = fruits.concat(programmingLanguages)
console.log(mixed)


let randomNum= Math.round(Math.random() * 100)
let guess = Number(prompt("Enter any number"))
while (randomNum != guess) {
    guess = Number(prompt("Wrong! try again: "))
}
console.log("Congrats!")


